tap_frontapp/streams.py

```diff
@@ -149,6 +149,126 @@ def sync_metric(atx, metric, incremental_range, start_date, end_date):
                     "num_composed_p": row[8]['p']
                     })
 
+    # transform the team_table data
+    if metric == 'tags_table':
+        for row in data:
+            data_rows.append({
+                "analytics_date": start_date_formatted,
+                "analytics_range": incremental_range,
+                "tag_v": row[0]['v'],
+                "tag_url": row[0]['url'],
+                "tag_id": row[0]['id'],
+                "conversations_archived_v": row[1]['v'],
+                "conversations_archived_p": row[1]['p'],
+                "conversations_open_v": row[2]['v'],
+                "conversations_open_p": row[2]['p'],
+                "conversations_total_v": row[3]['v'],
+                "conversations_total_p": row[3]['p'],
+                "num_messages_v": row[4]['v'],
+                "num_messages_p": row[4]['p'],
+                "avg_message_conversations_v": row[5]['v'],
+                "avg_message_conversations_p": row[5]['p']
+                })
+
+    # transform the customers_table data
+    if metric == 'customers_table':
+        for row in data:
+
+            # Some resource (ex. of type 'contact') don't have URLs
+            if not 'url' in row[0]:
+                row[0]['url'] = None
```

Comment:
Similar to the other PR, not a blocker but python's `dict.get(key, default_value)` could be used instead.

See https://docs.quantifiedcode.com/python-anti-patterns/correctness/not_using_get_to_return_a_default_value_from_a_dictionary.html#use-dict-get-key-default-to-assign-default-values